dags/sentinel-2_indexing.py

```diff
@@ -68,30 +94,38 @@
         name="datacube-index",
         task_id="indexing-task",
         get_logs=True,
+        is_delete_operator_pod=True,
     )
 
-    # TODO: implement
     UPDATE_RANGES = KubernetesPodOperator(
         namespace="processing",
         image=OWS_IMAGE,
         cmds=["datacube-ows-update"],
-        arguments=["--help"],
+        arguments=["s2_l2a"],
         labels={"step": "ows"},
         name="ows-update-ranges",
         task_id="update-ranges-task",
         get_logs=True,
+        volumes=[volume],
+        init_containers=[init_container],
+        is_delete_operator_pod=True,
```

Comment:
need to set the required env vars too:
```
      # Defaults are defined in .env file
      DB_HOSTNAME: ${DB_HOSTNAME}
      DB_PORT: ${DB_PORT}
      DB_USERNAME: ${DB_USERNAME}
      DB_PASSWORD: ${DB_PASSWORD}
      DB_DATABASE: ${DB_DATABASE}
      # Where the config file is (must match the volume mount)
      WMS_CONFIG_PATH: /env/config/ows_cfg.py
      # Path from the PYTHONPATH to the config object (default PYTHONPATH is /env) 
      DATACUBE_OWS_CFG: config.ows_cfg.ows_cfg
      AWS_DEFAULT_REGION: ${AWS_REGION}
      # Talk to AWS without using credentials
      AWS_NO_SIGN_REQUEST: "${AWS_NO_SIGN_REQUEST}"
```

Reply:
The `DEFAULT_ARGS` get passed in to all of them. The AWS-specific flags do need to be set, though.